src/Magneto/Core/CacheConfig.cs

```diff
@@ -43,7 +40,6 @@ public virtual object VaryBy
 		}
 		object _varyBy;
 
-		/// <inheritdoc />
 		public virtual string Key => _key ?? (_key = _createKey(KeyPrefix, VaryBy));
 
 		string buildKey(string keyPrefix, object varyBy)

```

Comment:
Not part of this PR, but get rid of the new List in this method, and do:
```csharp
return KeyPrefix + string.Join("_", VaryBy.Flatten());
```
for less allocations.

Reply:
Thanks. I've incorporated this suggested change.